src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.less

```diff
@@ -0,0 +1,117 @@
+@import 'base.less';
+
+.groups-create {
+  background-color: @main-color;
+  padding: 20px 21px 33px;
+  .shadow;
+
+  &__row {
+    margin-bottom: 25px;
+
+    &--top {
+      margin-top: 25px;
+    }
+  }
+
+  &__title {
+    font-size: 25px;
+    font-weight: normal;
+    color: @heading-bg;
+  }
+
+  &__title-input {
+    border: 1px solid @border-color;
+  }
+
+  &::ng-deep {
+    .ql-editor {
+      border: 1px solid @border-color;
+      border-top: none;
+      margin: 0 !important;
+      padding-top: 14px !important;
+      padding-bottom: 14px !important;
+      height: 62px !important;
+    }
+
+    dropzone > .dropzone.dz-wrapper.dz-single .dz-preview {
+      width: auto;
+      height: auto;
+    }
+  }
+}
+
+.subtitle {
+  font-size: 13px;
+  line-height: 22px;
+  color: @body-text-color;
+  margin-bottom: 3px;
+}
+
+.required {
+  &::after {
+    content: "*";
+    color: @validation-color;
+    margin-left: 2px;
+  }
+}
+
+.invalid {
+  box-shadow: 0 0 5px @validation-color;
+  outline-color: @validation-color;
+}
+
+.validation-errors {
+  list-style: disc;
+  color: @validation-color;
+  padding-left: 20px;
+}
+
+// Styles for Images component
+.images {
+  margin: 0 0 0 -2px;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+
+  &__item {
+    position: relative;
+    padding: 0 0 2px 2px;
+    flex-grow: 1;
+    max-width: 25%;
+    flex-shrink: 1;
+
+    &-name {
+      display: block;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      overflow: hidden;
+    }
+
+    .icon-close {
+      position: absolute;
+      top: 1px;
+      right: 1px;
+      width: 25px;
+      height: 25px;
+      padding: 7px;
+      text-align: center;
+      border-radius: 50%;
+      background: #fff;
+      box-shadow: 0 0 5px rgba(0, 0, 0, .5);
+      cursor: pointer;
+      color: #555;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+
+    img {
+      width: 100%;
+    }
+  }
+}
```

Comment:
Maybe move to global styles or component

Reply:
+